pkg/alertmanager/alertstore/configdb/store.go

```diff
@@ -15,6 +15,7 @@ const (
 
 var (
 	errReadOnly = errors.New("configdb alertmanager config storage is read-only")
+	errState    = errors.New("legacy configdb alertmanager storage does not support state")
```

Comment:
I think we haven't deprecated it yet.

```suggestion
	errState    = errors.New("configdb alertmanager storage does not support state persistency")
```